src/snapred/backend/dao/response/ArtificialNormResponse.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import BaseModel, ConfigDict
+
+from snapred.meta.mantid.WorkspaceNameGenerator import WorkspaceName
+
+
+class ArtificialNormResponse(BaseModel):
+    diffractionWorkspace: WorkspaceName
+
+    model_config = ConfigDict(
```

Comment:
We should probably pick one or the other format for specifying model configs.
I like the class based one [here](https://github.com/neutrons/SNAPRed/pull/463/files#diff-6e3f1fde214dcd92be764d5319b082b4ab00e219770736d0fb942b61c741c3b5R15-R17) but I think we use the above more often in the code

Reply:
Done. Updated